prometheus/promhttp/http.go

```diff
@@ -190,6 +193,17 @@ func HandlerForTransactional(reg prometheus.TransactionalGatherer, opts HandlerO
 		} else {
 			contentType = expfmt.Negotiate(req.Header)
 		}
+
+		if hasEscapedCollisions {
+			switch contentType.ToEscapingScheme() {
+			case model.UnderscoreEscaping, model.DotsEscaping:
+				opts.ErrorLog.Println("error: one or more metrics collide when escaped")
```

Comment:
need a better error message?

Reply:
maybe something like 

"One or more metrics collide when UTF-8 characters in name/label are translated to underscores"?